tests/models/pubsub.js

```diff
@@ -726,6 +726,30 @@ addTestWithRoundedTime(
   }
 );
 
+Tinytest.addAsync('Models - PubSub - Wait Time - track wait time', async (test, done) => {
```

Comment:
This test seems to be flaky. You probably could make it faster, and check that waitTime > 0.

Reply:
done!